### PR TITLE
Wrap all macros into one dropdown

### PR DESCRIPTION
**Pull request by wielinde**
getting rid of the icons and being more explanatory.

### CODE REVIEW

**Comment by wielinde** [comment]
@oliverguenther I don't understand why there are conflicting files. I have the latest editor, modified it, build it and committed it here. Why are there conflicts?

**Comment by oliverguenther** [comment]
@wielinde Because your merge base on dev is not up-to-date. I merged dev for you

**Review by oliverguenther — APPROVED** [review]
Looking good :+1:

File: spec/support/components/wysiwyg/wysiwyg_editor.rb [hunk 1]
```diff
@@ -50,5 +50,10 @@ def click_and_type_slowly(text)
 
       sleep 0.5
     end
+
+    def insert_macro(label)
```
Comment (oliverguenther) [inline spec/support/components/wysiwyg/wysiwyg_editor.rb:54]: :+1:

**Comment by oliverguenther** [comment]
Somehow travis did not correctly associate the last build result with this PR. It's green https://travis-ci.org/opf/openproject/builds/407015161

**Pull request merged**